Blog/blog/2022-09-16-introducing-seaography.md

```diff
@@ -0,0 +1,131 @@
+
+Anyone who are new to SeaORM, no worries, we have your back. You only need to provide a database connection, the `seaography-cli` will helps you generate the SeaORM entities from scratch together with all the steps mentioned above.
+
+## Design
+
+Lets "pop under the hood" and explore how this project is structured. The project is separated into the following crates:
+
+* [`seaography`](https://github.com/SeaQL/seaography): The library crate, it re-export all the macros and define helper structures and functions required to extend a SeaORM entity into a GraphQL node.
+
+* [`seaography-cli`](https://github.com/SeaQL/seaography/tree/main/cli): The CLI tool, it generates a ready to run GraphQL API server and SeaORM entities based on a user provided database URL. Code generation pipeline have two parts. It will first discover the database schema and represent it in a vendor agnostic way. Then, perform the actual code generation based on it.
+
+* [`seaography-discoverer`](https://github.com/SeaQL/seaography/tree/main/discoverer): A helper crate that will be used by the CLI tool to discover the database schema and parsing it as a vendor agnostic schema.
+
+* [`seaography-generator`](https://github.com/SeaQL/seaography/tree/main/generator): A helper crate that will be used by the CLI tool to consumes the vendor agnostic schema and generates the SeaORM entities and the required server boilerplate for the GraphQL API server.
+
+* [`seaography-derive`](https://github.com/SeaQL/seaography/tree/main/derive): A procedural macros crate responsible to derive types and implementations to integrate SeaORM entity with GraphQL types.
+
+### Supported Features
+
+* Relational query (1-to-1, 1-to-N)
+* Pagination on query's root entity
+* Filter with operators (e.g. gt, lt, eq)
+* Order by any column
+
+Right now there is no mutation, but it's on our plan!
+
+## Getting Started
+
+To quick start, we have the following examples for you, alongside with the SQL scripts to initialize the database.
+
+* [MySQL](https://github.com/SeaQL/seaography/tree/main/examples/mysql)
+* [PostgreSQL](https://github.com/SeaQL/seaography/tree/main/examples/postgres)
+* [SQLite](https://github.com/SeaQL/seaography/tree/main/examples/sqlite)
+
+All examples provide a web-based GraphQL playground when running, on it you can inspect the GraphQL schema and query data. We also hosted a [demo GraphQL playground](https://playground.sea-ql.org/seaography) in case you can't wait to play with it.
+
+If you wish to start a GraphQL server for your own database. Please read on.
+
+* [Extending SeaORM entity to support GraphQL queries](https://www.sea-ql.org/Seaography/docs/getting-started/)
+* [Generating from scratch with an existing database](https://www.sea-ql.org/Seaography/docs/getting-started/)
```

Comment:
Hey @karatakis, thanks for the draft!! Please check the blog post and let me know your thoughts. I've edit quite a lot.

I think the "Example" part contain too much details and we should move it into the documentation. In the blog we can point reader back to the documentation if they want to read on.

What do you think?